src/clusterfuzz/_internal/tests/core/bot/tasks/utasks/uworker_io_test.py

```diff
@@ -169,7 +169,7 @@ def test_error_and_testcase_behavior(self):
     error_value = 1
     self.output.error_type = error_value
     self.assertEqual(self.output.error_type, error_value)
-    self.assertEqual(self.output.proto.error_type, error_value)
+    self.assertEqual(self.output._proto.error_type, error_value)  # pylint: disable=protected-member
```

Comment:
You could instead:

- just add a `proto()` method to `UworkerMsg`, or even a `proto_copy()` if you really want to avoid exposing internal state
- roundtrip `self.output` and check that `error_type` is still good